config/initializers/razorpay.rb

```diff
@@ -0,0 +1,5 @@
+require "razorpay"
+
+# key_id = Rails.application.credentials.dig(:razorpay, :key_id)
+# secret_key = Rails.application.credentials.dig(:razorpay, :secret_key)
+Razorpay.setup('rzp_test_7uFpRekBxdblL5', 'Uabud32eND8nHlqhNxu0mUN2')
```

Comment:
These should not be committed, use encrypted credentials!

Reply:
To encrypt the credentials we will need the master encryption key.
Can you please provide master encryption key?

Reply:
@yashmtrivedi - we sent the development.key to JPM a while back, are you not able to access it? If not, do let me know and I'll send it across again. I'll need the email to share it with.

Reply:
@mikevic - that development key is not working. 
Error: Couldn't decrypt config/credentials.yml.enc. Perhaps you passed the wrong key?
this is the error i am getting when i use the development.key.
Can you please resend the key on yash.m.trivedi@tsgforce.com .

Reply:
@yashmtrivedi - could you try to decrypt - development.yml.enc rather than credentials.yml.enc

Let me know if that works?